scripts/generate_driver_wrappers.py

```diff
@@ -176,24 +176,31 @@ def main() -> int:
                         help='output file\'s location')
     args = parser.parse_args()
 
-    mbedtls_root = os.path.abspath(args.mbedtls_root)
+    repo_root = os.path.abspath(args.repo_root)
+
+    library_dir = ''
+    if build_tree.looks_like_mbedtls_root(repo_root):
+        library_dir = 'library'
+    elif build_tree.looks_like_tf_psa_crypto_root(repo_root):
+        library_dir = 'core'
```

Comment:
```suggestion
    library_dir = build_tree.crypto_core_directory(repo_root)
```

Reply:
I was going to change this, the reason I hadn't yet was because I was waiting for feedback on whether `build_tree.crypto_core_directory()` was the implementation we wanted. Once I get confirmation on this I think there will be a few replacements to make elsewhere.